aten/src/ATen/core/op_registration/op_registration.cpp

```diff
@@ -134,7 +134,7 @@ void RegisterOperators::registerOp_(FunctionSchema&& schema, Options&& options)
 }
 
 void RegisterOperators::registerSchemaAndKernel_(FunctionSchema schema, Options::KernelRegistrationConfig&& kernel) {
-  AT_ASSERTM(kernel.kernel_func != nullptr && static_cast<bool>(kernel.cache_creator_func), "Kernel must be set");
+  TORCH_INTERNAL_ASSERT(kernel.kernel_func != nullptr && static_cast<bool>(kernel.cache_creator_func), "Kernel must be set");
```

Comment:
Similarly here

Reply:
`registerSchemaAndKernel_` is a private method. If, by using public methods, you're triggering this assert, then that is a pytorch bug.